src/app/app.module.ts

```diff
@@ -7,6 +7,8 @@ import { LoginComponent } from './login/login.component';
 import { AppComponent } from './app.component';
 import { RegistrationComponent } from './registration/registration.component';
 import { HomepageComponent } from './homepage/homepage.component';
+import { SongsComponent } from './song/songs.component';
+import { SongComponent } from './song/song.component';
```

Comment:
Don't forget the following:
`import { SongService } from './shared/services/song.service';`